tasks/managed/collect-index-images/collect-index-images.yaml

```diff
@@ -0,0 +1,66 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: collect-index-images
```

Comment:
Sorry to be holding this back once more... But 2 days ago this got merged: https://github.com/konflux-ci/release-service-catalog/pull/1048

The whole pipeline is now using trusted artifacts. So this new task will have to support it as well - the shared workspace will no longer be available when run in Konflux. Unfortunately I don't have much details on that in our contributing guide yet: https://github.com/konflux-ci/release-service-catalog/blob/development/CONTRIBUTING.md

There is a video that explains how it works: https://miro.com/app/board/uXjVIbidSuI=/?playRecording=5ed2a205-bad5-4f62-a1b1-1dae27ddc65c

But otherwise you will have to see other tasks and copy from there. If you need more help, you can contact Scott Hebert.

Reply:
Updated to support trusted artifacts.

Reply:
Thanks. I can see that the unit tests ran successfully with trusted artifacts enabled, so that looks good. @scoheb can you take a look as well please?

Reply:
I think this looks good - I assume you copied the approach from some other tasks, which is what I would do as well. And I checked that the source artifact for the create-pyxis-image task is the right one.